### ISSUE TITLE
Bug introduced with 2.12 for annotate method

### ISSUE THREAD

**Issue by johnvblazic** [clulab/processors#133]
Mihai gave us some example code for class using this library the day after 2.12 released. I realize you haven't tuned it for 2.12 yet, but I found an issue with calling the annotate method that needs to be fixed when you update it for the new version. The scala code used below works flawlessly with scala 2.11.8, but fails when the build.sbt is updated to what you see below, using 2.12 and the processors library for 2.11. It seems a method was renamed in 2.12 that annotate is using. See the error message at the bottom

**Build.sbt:**

```
name := "Scala_Testing"

version := "1.0"

scalaVersion := "2.12.1"

libraryDependencies ++= Seq(
  "org.clulab" % "processors-corenlp_2.11" % "6.0.1",
  "org.clulab" % "processors-main_2.11" % "6.0.1",
  "org.clulab" % "processors-models_2.11" % "6.0.1"
)
```    

**Scala code used:**

```
package edu.arizona.cs

import scala.io.Source
import org.clulab.processors.fastnlp.FastNLPProcessor

/**
  * Created by johnblazic on 1/20/17.
  */
class FirstScala(val fileToProcess:String, val dummy:Int =0) {
  //def this(val ftp:String) {this(ftp, 0)}

  //val processor = new FastNLPProcessor()

  def process() {
    val processor = new FastNLPProcessor()
    //println("Hello World!")
    val text = Source.fromFile(fileToProcess).getLines.mkString(" ")
    println(text)

    val doc = processor.annotate(text)

    for(s <- doc.sentences){
      val words =  s.words
      val tags = s.tags.get
      val nouns = words.zip(tags).filter(_._2.startsWith("NN")).map(_._1)
      println("Words: " + s.words.mkString(", "))
      println("Nouns: " + nouns.mkString(", "))
    }
  }
}

object FirstScala {
  def main(args:Array[String]) {
    val fs = new FirstScala(args(0))
    fs.process()
  }
}
```

**Error message:**
[error] (run-main-0) java.lang.NoSuchMethodError: scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
java.lang.NoSuchMethodError: scala.Predef$.refArrayOps([Ljava/lang/Object;)Lscala/collection/mutable/ArrayOps;
	at org.clulab.processors.shallownlp.ShallowNLPProcessor.postprocessTokens(ShallowNLPProcessor.scala:88)
	at org.clulab.processors.shallownlp.ShallowNLPProcessor.mkSentence(ShallowNLPProcessor.scala:127)
	at org.clulab.processors.shallownlp.ShallowNLPProcessor$$anonfun$mkDocument$1.apply(ShallowNLPProcessor.scala:107)
	at org.clulab.processors.shallownlp.ShallowNLPProcessor$$anonfun$mkDocument$1.apply(ShallowNLPProcessor.scala:106)
	at scala.collection.Iterator.foreach(Iterator.scala:929)
	at scala.collection.Iterator.foreach$(Iterator.scala:929)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1406)
	at scala.collection.IterableLike.foreach(IterableLike.scala:71)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at org.clulab.processors.shallownlp.ShallowNLPProcessor.mkDocument(ShallowNLPProcessor.scala:106)
	at org.clulab.processors.Processor$class.annotate(Processor.scala:87)
	at org.clulab.processors.shallownlp.ShallowNLPProcessor.annotate(ShallowNLPProcessor.scala:25)
	at edu.arizona.cs.FirstScala.process(FirstScala.scala:19)
	at edu.arizona.cs.FirstScala$.main(FirstScala.scala:34)
	at edu.arizona.cs.FirstScala.main(FirstScala.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)

**Comment by hickst** [comment]
The Processors library has now been "officially" switched to use Scala 2.12 by default. Could you please retry your code to see if this issue still exists. Thanks.

**Comment by myedibleenso** [comment]
@hickst, will you please update CHANGES to reflect the cross compile for 2.12?  I had just checked  there wondering if that had been merged or not.

@johnvblazic, we haven't published a 2.12 version to maven yet, so you'll need to test by publishing the library locally.

**Comment by hickst** [comment]
The CHANGES file was updated as part of the Processors 7.0.0 release. Once 7.0.0 reaches Maven Central (in a few hours) you will **not** have to publish locally (just use 7.0.0).
